lib/api/route/realm.dart

```diff
@@ -34,7 +34,8 @@ class GetServerSettingsResult {
 
   final int zulipFeatureLevel;
   final String zulipVersion;
-  final String? zulipMergeBase; // TODO(server-5)
+  // TODO: Modernize this once we get to #267
+  final String? zulipMergeBase;
```

Comment:
This should keep a `TODO(server-5)` comment until we do actually modernize it. That way it naturally shows up in any future round of grepping for opportunities to modernize our API usage.